pkg/api/v1beta3/dynakube/kspm/props.go

```diff
@@ -11,3 +11,7 @@ func (kspm *Kspm) IsEnabled() bool {
 func (kspm *Kspm) GetTokenSecretName() string {
 	return kspm.name + "-" + TokenSecretKey
 }
+
+func (kspm *Kspm) GetDaemonSetName() string {
+	return kspm.name + "-" + NodeCollectorNameSuffix
```

Comment:
pls rewrite this as the resulting name of the ds is:

`dynakube-test--node-config-collector`

Reply:
fixed